lib/teracy-dev-essential/config/save_mac_address.rb

```diff
@@ -5,12 +5,48 @@ module TeracyDevEssential
   module Config
     class SaveMacAddress < TeracyDev::Config::Configurator
       def configure(settings, config, type:)
```

Comment:
there is nothing to do with common, so please override: `configure_node` instead.